### PR TITLE
add explaining Intro sentence to examples

### PR DESCRIPTION
**Pull request by testbird**

### CODE REVIEW

File: src/pages/main_layout.cr [hunk 1]
```diff
@@ -109,7 +109,13 @@ abstract class MainLayout
     If this sounds interesting, [let's get started](#{Guides::GettingStarted::Installing.path}).
 
     ## What does Lucky look like?
-
+    
+    In Lucky we generate separate action classes with names that describe the route,
+    and they define the routing paths together with the response.
```
Comment (paulcsmith) [inline src/pages/main_layout.cr]: This second part "and they define the routing paths" sounds similar to "that describe the route". So I think "and they define the routing paths" can be removed

**Review by paulcsmith — COMMENTED** [review]
Just one small comment and then ready to go!

File: src/pages/main_layout.cr (hunk 1 above)
Comment (testbird) [reply src/pages/main_layout.cr]: I already wanted to hint at the generator working based on the name.

Maybe this:
> In Lucky we generate separate action classes with names that describe the route.
> The resulting file maps routing path definitions to a response block.

Though, currently the paths may still be implicit, the mapping is there.

File: src/pages/main_layout.cr [hunk 2]
```diff
@@ -110,8 +110,8 @@ abstract class MainLayout
 
     ## What does Lucky look like?
     
-    In Lucky we generate separate action classes with names that describe the route,
-    and they define the routing paths together with the response.
+    In Lucky we generate separate action classes with names that describe the route.
+    The resulting file maps routing path definitions to a response block.
```
Comment (paulcsmith) [inline src/pages/main_layout.cr:114]: Looks great!

**Pull request merged**